nursery/capture-webcam-video.yml

```
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: capture webcam video
+    namespace: collection/webcam
+    author: "@johnk3r"
+    description: Rule that detects a system's webcam being used to capture video
+    scope: function
+    att&ck:
+      - Collection::Video Capture [T1125]
+  features:
+    - and:
+      - os: windows
+      - api: capCreateCaptureWindow
+      - basic block:
+        - and:
+          - api: SendMessage
+          - number: 0x43E = WM_CAP_SEQUENCE
+      - or:          
```

```suggestion
      - or:
```

cleanup whitespace to make the linter happy and then i'll merge!